### DISCUSSION TITLE
Link to open specific lightbox

### DISCUSSION THREAD

**Discussion by janstieler** [biati-digital/glightbox#360] [Q&A] [answered]
Hi,
how can I set a second link to a specific lightbox.
If I use the same class with `.lightbox` I get two slides with the same content.

Cheers

**Comment by gingerchew** [comment, marked as answer]
I don't think its possible right now without JS intervention. Something like [this](https://codepen.io/gingerchew/pen/NWMWwzE) with the `.openAt(index)` method makes it possible.

**Reply by janstieler** [reply]
Hi, thanks this solution works like a charme!
